src/ui/views/EditAwards.basketball.tsx

```diff
@@ -0,0 +1,329 @@
+import React, { FormEvent, useState } from "react";
+
+import useTitleBar from "../hooks/useTitleBar";
+import type { Player, View } from "../../common/types";
+import { logEvent, toWorker, helpers, realtimeUpdate } from "../util";
+import SelectReact from "../components/SelectMultiple";
+const EditAwardsBasketball = ({
+	godMode,
+	players,
+	awards,
+	season,
+}: View<"editAwards">) => {
+	useTitleBar({
+		title: "Edit awards",
+		dropdownView: "edit_awards",
+		dropdownFields: { seasons: season },
+	});
+	const awardsInitial = awards;
+	const [state, setState] = useState(() => {
```

Comment:
Since there's only one property in your `state` object, it would be more idiomatic to do:

    const [aws, setAws] = useState(() => helpers.deepCopy(awards));

Also that deepCopy is a code smell, ideally you would never be mutating that object. But I understand sometimes it's necessary for convenience, I haven't read enough of the rest of this function to see if that makes sense here :)